@fetch-mock/vitest doesn't works with esm package

I try set up fetch-mock/vitest with ESM enabled, it error because no extension found on import path.

```
Error: Cannot find module 'node_modules/.pnpm/@fetch-mock+vitest@0.2.2_vitest@2.1.3/node_modules/@fetch-mock/vitest/dist/esm/vitest-extensions' imported from node_modules/.pnpm/@fetch-mock+vitest@0.2.2_vitest@2.1.3/node_modules/@fetch-mock/vitest/dist/esm/index.js
```

Reproducible repository: https://stackblitz.com/edit/fetch-mock-repro-esm-error?file=README.md&view=editor

The above may seem like a lot of work, but the easier you make it for me to 
recreate the bug, the sooner it'll get fixed.

--- a/jest.config.js
+++ b/jest.config.js
@@ -6,5 +6,6 @@ export default {
 	},
 	moduleNameMapper: {
 		'@fetch-mock/core': '<rootDir>/packages/core/dist/cjs/index.js',
+		'(.+)\\.js': '$1',
 	},
 };

--- a/packages/jest/src/index.ts
+++ b/packages/jest/src/index.ts
@@ -3,7 +3,7 @@ import {
 	defaultFetchMockConfig,
 	RemoveRouteOptions,
 } from 'fetch-mock';
-import './jest-extensions';
+import './jest-extensions.js';
 import type { Jest } from '@jest/environment';
 
 type MockResetOptions = {

--- a/packages/vitest/src/index.ts
+++ b/packages/vitest/src/index.ts
@@ -4,7 +4,7 @@ import {
 	defaultFetchMockConfig,
 	RemoveRouteOptions,
 } from 'fetch-mock';
-import './vitest-extensions';
+import './vitest-extensions.js';
 
 type MockResetOptions = {
 	includeSticky: boolean;

